src/com/jalasoft/search/view/Table.java

```diff
@@ -1,41 +1,29 @@
 package com.jalasoft.search.view;
 
-import javax.swing.*;
-import java.awt.*;
+import javax.swing.JPanel;
+import javax.swing.JTable;
+import javax.swing.JScrollPane;
+import java.awt.GridLayout;
+import java.awt.Dimension;
 
 public class Table extends JPanel {
 
+    private String [] columnNames;
+    private Object [] [] data;
```

Comment:
If you declare the variable"data" as private then I will not be able to access it unless you have a get method of this variable